documentation/src/test/java/example/AssertionsDemo.java

```diff
@@ -143,23 +143,3 @@ private static String greeting() {
 }
 // end::user_guide[]
 // @formatter:on
-
-class Person {
```

Comment:
I couldn't access this class in `AssertionsDemoKotlin` without making it public. It was just easier to make `Person` a kotlin data class that both tests use.